packages/kit/src/exports/vite/preview/index.js

```diff
@@ -36,6 +36,10 @@ export async function preview(vite, vite_config, svelte_config) {
 
 	const dir = join(svelte_config.kit.outDir, 'output/server');
 
+	if (!existsSync(dir)) {
```

Comment:
```suggestion
	if (!fs.existsSync(dir)) {
```

We're already importing all of `node:fs`, we should probably just use it directly.